ui/imports/shared/views/chat/ChatTextView.qml

```diff
@@ -0,0 +1,180 @@
+                        )
+                {
+                    root.store.chatsModelInst.channelView.setActiveChannel(channelName);
+                }
+
+                return
+            }
+
+            if (link.startsWith('//')) {
+                let pk = link.replace("//", "");
+                const userProfileImage = appMain.getProfileImage(pk)
+                openProfilePopup(root.store.chatsModelInst.userNameOrAlias(pk), pk, userProfileImage || root.store.utilsModelInst.generateIdenticon(pk))
+                return;
+            }
+
+            const data = Utils.getLinkDataForStatusLinks(link)
+            if (data && data.callback) {
+                return data.callback()
+            }
+
+
+            appMain.openLink(link)
+        }
+
+        onLinkHovered: {
+            cursorShape: Qt.PointingHandCursor
+        }
+
+        text: {
+            if(contentType === Constants.stickerType) return "";
+            let msg = Utils.linkifyAndXSS(message);
+            if(isEmoji) {
+                return Emoji.parse(msg, Emoji.size.middle);
+            } else {
+                if(isEdited){
+                    let index = msg.endsWith("code>") ? msg.length : msg.length - 4
+                    return Utils.getMessageWithStyle(Emoji.parse(msg.slice(0, index) + Constants.editLabel + msg.slice(index)), localAccountSensitiveSettings.useCompactMode, isCurrentUser, hoveredLink)
+                }
+                return Utils.getMessageWithStyle(Emoji.parse(msg), localAccountSensitiveSettings.useCompactMode, isCurrentUser, hoveredLink)
+            }
+        }
+    }
+
+    Loader {
+        id: mask
+        anchors.fill: chatText
+        active: showMoreLoader.active
+        visible: false
+        sourceComponent: LinearGradient {
+            start: Qt.point(0, 0)
+            end: Qt.point(0, chatText.height)
+            gradient: Gradient {
+                GradientStop { position: 0.0; color: "white" }
+                GradientStop { position: 0.85; color: "white" }
+                GradientStop { position: 1; color: "transparent" }
+            }
+        }
+    }
+
+    Loader {
+        id: opMask
+        active: showMoreLoader.active && !root.readMore
+        anchors.fill: chatText
+        sourceComponent: OpacityMask {
+            source: chatText
+            maskSource: mask
+        }
+    }
+
+    Loader {
+        id: showMoreLoader
+        active: root.veryLongChatText
+        anchors.top: chatText.bottom
+        anchors.topMargin: - Style.current.padding
+        anchors.horizontalCenter: parent.horizontalCenter
+        sourceComponent: Component {
+            SVGImage {
+                id: emojiImage
+                width: 256
+                height: 44
+                fillMode: Image.PreserveAspectFit
+                source: Style.svg("read-more")
+                z: 100
+                rotation: root.readMore ? 180 : 0
+                MouseArea {
+                    z: 101
```

Comment:
maybe its not your code, but... its necessary to up mouse area to 101 in this svg component? This `z` affecting just on siblings which not exists :)

Reply:
removed